docs/src/reference/cli/changelog.rst

```diff
@@ -1,9 +1,13 @@
 CLI Changelog
 =============
 
+.. _cli-21.2.0:
```

Comment:
As mentioned in the PR body, I'm hoping to someday be able to automate the pulling in of the packages' changelogs rather than needing to manually add these refs and `start-*` & `end-*` tags. But I just wanted to get this out for now and will otherwise document the process.